app/models/video_watch.rb

```diff
@@ -0,0 +1,4 @@
+class VideoWatch < ApplicationRecord
```

Comment:
I think the Table name should be Video

Reply:
Apu, I have a attached content named `videos`. To distinguish this it was named `VideoWatch` which is storing the video watch history of users.